dbms/src/Server/Server.cpp

```diff
@@ -1049,6 +1049,25 @@ int Server::main(const std::vector<std::string> & /*args*/)
         LOG_FMT_INFO(log, "tiflash proxy thread is joined");
     });
 
+    /// get server info.
+    {
+        diagnosticspb::ServerInfoRequest request;
+        request.set_tp(static_cast<diagnosticspb::ServerInfoType>(1));
+        diagnosticspb::ServerInfoResponse response;
+        auto * helper = tiflash_instance_wrap.proxy_helper;
+        if (helper)
+        {
+            std::string req = request.SerializeAsString();
+            helper->fn_server_info(helper->proxy_ptr, strIntoView(&req), &response);
+        }
+        else
+        {
+            LOG_FMT_INFO(log, "TiFlashRaftProxyHelper is null, failed to get server info");
+        }
+        server_info.parseSysInfo(response);
+        LOG_FMT_INFO(log, "ServerInfo: {}", server_info.debugString());
```

Comment:
Do we use these infos for our threading/limiter etc rather than just logging?

Reply:
will use `logical cpu cores` to determine the default background thread pool size. Other variables also can be used, but I have no plan so far.